scipy/special/_basic.py

```diff
@@ -3071,26 +3102,35 @@ def factorialk(n, k, exact=True):
 
       factorialk(n, 2) = factorial2(n)
 
+    While less straight-forward than for the double-factorial, it's possible to
+    calculate a general approximation formula of n!(k) by studying ``n`` for a given
+    residue ``r < k`` (thus ``n = m * k + r``, resp. ``r = n % k``), which can be
```

Comment:
I can do that, though the terms mean the same thing (at least etymologically). In my math education, the nomenclature w.r.t. equivalence relations was always "residue" / "residual" (though "remainder" is common when used with division explicitly).

Reply:
Yeah, I know they mean the same thing. I'm just thinking in terms of accessibility. I didn't encounter residue used this way in a class until the 3rd year of my math undergraduate program, but I've known what a remainder is since primary school.